### DISCUSSION TITLE
Is there any way to record the steering value without using the recording API?

### DISCUSSION THREAD

**Discussion by joshuapatterson797** [microsoft/AirSim#4159] [Support Q&A]
I'm working on a project which requires that I get the steering angle from the car in AIRsim, but the recording API doesn't seem to have the customizability to record the steering value alone.  Is there any alternative ways to do this? I've looked everywhere online, but I can't seem to find anything about the subject.

**Comment by zimmy87** [comment]
You could call the `getCarControls` API in a loop and record the `CarControls.steering` value returned.
